projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/order-history.e2e.cy.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { viewportContext } from '../../helpers/viewport-context';
+
+describe('Order History Page accessibility', { testIsolation: false }, () => {
+  viewportContext(['mobile', 'desktop'], () => {
```

Comment:
```suggestion
describe('Order History Page accessibility', { testIsolation: false }, () => {
```

If I'm correct there is no difference between mobile and desktop content. If that's the case no need to test it twice. 

I can see having the test isolation off makes the test much faster, maybe we should encourage this in all our Continuum tests?